draft-ietf-rats-eat.md

```diff
@@ -1099,12 +1103,53 @@ string naming the submodule. No submodules may have the same name.
 {::include cddl/submods.cddl}
 ~~~~
 
+
 # Endorsements and Verification Keys {#keyid}
 
 TODO: fill this section in. It will discuss key IDs, endorsement ID and such that
 are needed as input needed to by the Verifier to verify the signature. This will
 NOT discuss the contents of an Endorsement, just and ID/locator.
 
+
+# Layered Attestation {#layered}
+
+It is possible for a device to have multiple Attesting Environments as described in {{RATS-Architecture}}.
+When these are layered, one is a root the endorsement and/or verification key originates with the device manufacturer or endorser.
+The trust for those that are layered above the root comes from Attestation Evidence about them that is provided by the root or a layer below.
+
+A layered Attesting Environment is always described as a submodule.
+To know that a particular submodule is providing evidence for an Attesting Environment, it must have an intended use claim with the value of "attest=env".
+See {{intended-use}}.
+
+The submodule should also contain a claim that hold the verification key, the key used to verify signed attestation evidence from the layered attester.
```

Comment:
That's impossible for me to parse. The current Attesting Environment should include the pub-key of a layered Attesting Environment in the Evidence generated about that layered Attesting Environment? If so, maybe elaborate on the reasoning? What's the use case you have in mind here?

Reply:
1121 seems too strong. The names of the measurements pertaining to the attesting environment (or other measurements in the target) provides intended use semantics. If using submods then submod-name captures the intended semantics.

1124 seems to be saying layers can be cryptographically linked, therefore it may be appropriate to include a public key in the Evidence related to an attesting environment.